librz/util/subprocess.c

```diff
@@ -632,7 +620,7 @@ static char **create_child_env(const char *envvars[], const char *envvals[], siz
 	for (size_t i = 0; i < size; i++) {
 		new_env[i] = strdup(environ[i]);
 	}
-	for (size_t i = 0; i <= env_size; i++) {
+	for (size_t i = 0; i <= new_env_size; i++) {
```

Comment:
nice catch... I guess it was wrong before.

Reply:
i had a huge headache on this since asan was complaining but not telling me where due optimizations.

Reply:
compile with `-buildtype=debug` if that was the problem.